ui/app/src/components/ItemDisplay/ItemDisplay.tsx

```diff
@@ -124,12 +125,7 @@ const ItemDisplay = forwardRef<HTMLSpanElement, ItemDisplayProps>((props, ref) =
 						/>
 					)}
 			{showItemType && (
-				<ItemTypeIcon
-					{...itemTypeIconProps}
-					item={item}
-					className={[classes?.icon, itemTypeIconProps?.className].filter(Boolean).join(' ')}
-					sx={{ fontSize: '1.1rem', ...sxs?.icon }}
-				/>
+				<DisabledItemIcon item={item} itemTypeIconProps={itemTypeIconProps} sxs={sxs} classes={classes} />
```

Comment:
The idea was to show the encapsulate the disabled icon logic and show that component if item is disabled; not to obscurely wrap all icons under a component called DisabledIcon